plugins/colorbutton/plugin.js

```diff
@@ -331,31 +359,47 @@ CKEDITOR.plugins.add( 'colorbutton', {
 					colorLabel = colorName;
 				}
 
-				output.push( '<td>' +
-					'<a class="cke_colorbox" _cke_focus=1 hidefocus=true' +
-						' title="', colorLabel, '"' +
-						' draggable="false"' +
-						' ondragstart="return false;"' + // Draggable attribute is buggy on Firefox.
-						' onclick="CKEDITOR.tools.callFunction(', clickFn, ',\'', colorCode, '\',\'', type, '\'); return false;"' +
-						' href="javascript:void(\'', colorCode, '\')"' +
-						' data-value="' + colorCode + '"' +
-						' role="option" aria-posinset="', ( i + 2 ), '" aria-setsize="', total, '">' +
-						'<span class="cke_colorbox" style="background-color:#', colorCode, '"></span>' +
-					'</a>' +
-					'</td>' );
+				output.push( '<td>' + generateTileHtml( {
+					colorLabel: colorLabel,
+					clickFn: clickFn,
+					colorCode: colorCode,
+					type: type,
+					position: i + 2,
+					setSize: total
+				} ) + '</td>' );
 			}
 
-			// Render the "More Colors" button.
+			// Render the "Content Colors" section.
+			output.push( '</tr>' +
+				'<tr>' +
+					'<td colspan=' + colorsPerRow + '" align="center">' +
+						'<span class="cke_colorcontent_label" style="display:none">Content Colors</span>' +
```

Comment:
You should already start to using lang strings here instead of direct name `Content Coloros`. It is really easy to forget later about such cases.
To do so you need to make few things:
1. Add required fields to file `plugins/colorbutton/lang/en.js`
2. Run our langtools: `dev/langtool/langtool.sh`
3. This should modify js files in other languages for colorbutton with `// MISSING` comment
4. Run `dev/langtool/update_meta_files.sh` what should update meta file `dev/langtool/meta/ckeditor.plugin-colorbutton/meta.txt`
5. In this txt file should appear empty entries, where is required to describe your new strings for the translators.

And the same for the `Custom Colors`.

Reply:
Actually to save space in panel (and labels are no longer needed) I replaced names with `<hr>`, so maybe there will be no need to add any translations.

Reply:
Actually to save space in panel (and labels are no longer needed) I replaced names with `<hr>`, so maybe there will be no need to add any translations.